pkg/backend/filestate/backend.go

```diff
@@ -249,35 +250,35 @@ func (b *localBackend) Preview(ctx context.Context, stackRef backend.StackRefere
 }
 
 func (b *localBackend) Update(ctx context.Context, stackRef backend.StackReference,
-	op backend.UpdateOperation) (engine.ResourceChanges, error) {
+	op backend.UpdateOperation) (engine.ResourceChanges, *result.Result) {
 	stack, err := b.GetStack(ctx, stackRef)
 	if err != nil {
-		return nil, err
+		return nil, result.FromError(err)
```

Comment:
there's a reasonable argument to be made that .FromError should check for nil and either panic if misued, or return a nil-result.  Otherwise, if you pass a nil-error, you'll get a `Bail` result, which seems like something that should happen because the caller explicitly asked for that.